backendless_chef.yaml

```diff
@@ -578,11 +578,9 @@ Resources:
         Version: "2012-10-17"
         Statement:
           - Effect: "Allow"
-            Principal: "*"
+            Principal:
+              AWS: !GetAtt ChefRole.Arn
```

Comment:
Yeah, according to the docs it needs the ARN.  Updated to account for a user-provided IAM role.